<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidgetFlatten</class>
 <widget class="QWidget" name="SettingsWidgetFlatten">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsWidgetFlatten</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QCheckBox" name="flattened_check">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
